### ISSUE TITLE
Install with patch supported?

### ISSUE THREAD

**Issue by twnaing** [richin13/asdf-neovim#15]
Does this plugin support installing neovim with patch? 

asdf-python supports [Install with `--patch`](https://github.com/danhper/asdf-python/commit/225091e2eec76a911bfa778adc826c603c51644b).

**Comment by richin13** [comment]
Hello 👋🏼 

I believe it partially supports it. You can install any given branch or even a specific commit:

https://github.com/richin13/asdf-neovim/blob/1fb2f664449c57fa9b236094b947eb2e1d969ddf/bin/utils.sh#L45-L48

It should work as long as the url `https://github.com/neovim/neovim/archive/${version}.tar.gz` exits for whatever value for `${version}` you choose 👍🏼

**Comment by richin13** [comment]
Hey @twnaing, not sure if this is the functionality you were looking for but I discovered there was a bug when trying to install neovim using a specific commit. The bug's was fixed in 5b19ea5 👍🏼 

Usage:

```bash
asdf install neovim ref:1f1863ed54355c314a5016560809e65c3fb0cb79
```

(Also pushing closed this automatically, feel free to re-open it the resolution doesn't satisfy you)

**Comment by twnaing** [comment]
@richin13 I think it is not. This feature let user install any (specific commit or stable) with provided patch.

If you look at the link I provided, you will better understand, I think.
